pkgs/by-name/ni/nix-web/package.nix

```diff
@@ -3,24 +3,26 @@
 , fetchFromGitea
 , pkg-config
 , openssl
-, nix
+, nixVersions
+, nixVersion ? "nix_2_17"
```

Comment:
This virtually prevent from passing any arbitrary Nix version for test, I find it a bit sad.
Could we have something like `{ nixVersions, nix ? nixVersions.nix_2_17 }` rather ? More than often, a user will be able to pass the full package directly.

Reply:
I actually tried doing this at first, but an input called `nix` gets passed the default nix version, so I started doing this... but actually, I can just call it `nixPackage ? nixVersions.nix_2_17`. How's that?

Reply:
Seems fine to me for now.

Reply:
Yeah, let's revisit this later!

Reply:
Whoops, I forgot to actually make this change

Reply:
Feel free to send a follow up PR, I didn't want you to be blocked on those considerations :)